rmd/Story 1.json

```diff
@@ -616,393 +616,393 @@
   },
   "101010020120051": {
     "OriginalText": "フィルディア隊長！ 緊急事態です！",
-    "Text": "",
-    "Enabled": false
+    "Text": "Captain Fildia! There's an emergency!",
+    "Enabled": true
   },
   "101010020130004": {
     "OriginalText": "どうした？",
-    "Text": "",
-    "Enabled": false
+    "Text": "What happened?",
+    "Enabled": true
   },
   "101010020140051": {
     "OriginalText": "先遣隊がダーカーと接触、\n交戦を開始したのですが、\nフォトンが！ フォトンが使えません！",
-    "Text": "",
-    "Enabled": false
+    "Text": "The survey team has encountered a group of Darkers!\nAfight broke out but...\nThey- they can't use Photon!
```

Comment:
Missing space between A and fight
Photon should be Photons
Missing ", at the end of the text string, causing a script error